### ISSUE TITLE
Enemies that hold wands, often do not drop them after death. Bridge boss spawns like 20 wands in one spot when attacking.

### ISSUE THREAD

**Issue by xTartine** [IntQuant/noita_entangled_worlds#231]

**Comment by SwiftestPotato** [comment]
I'd like to confirm that this is in fact an ongoing issue. 

Several enemies that picked up wands before me due to reaching it first caused them to disappear upon their death. There seems to be an exception when I pick the wands up first and then drop them for an enemy to pick up, so the whole thing may revolve around who picks up the wand first but I've yet to test that theory properly. It seems to be a pattern on my runs, though. I'm thinking of downgrading to a previous version because enemies swallowing wands like a black hole upon their death is quite a predicament. 

I've also had issues today with not being able to give items, wands and spells I've picked up directly to my friend. I joined after he played for a while, and despite me being able to see the stuff he dropped, he couldn't see the stuff I've picked up or dropped. I could modify his wand with spells I've picked up and drop it to give him spells, but if I dropped anything I picked up first then it wouldn't show it on his end when they're dropped. 

Not sure if this is linked to the enemies' normally dropped held items disappearing, but it's another instance of dropped items seemingly not being rendered and being inaccessible. Though it may just be an issue on my (or my friend's) end.
